includes/core/class-wc-payments-customer-service-api.php

```diff
@@ -0,0 +1,183 @@
+<?php
```

Comment:
Shall we put this file or service into a dedicated folder? I would like to use this approach to prevent putting plenty of classes into root core directory. I would put it inside service folder? What do you think about this approach?

Reply:
We can do that, but didn't want to over-optimize too soon as I think we are currently planning for only two classes used by external consumers - Account_Service and Customer_Service. I think we can move it later, as the need arises?

Reply:
The way we structure now, someone will follow it and use it in the future. We could remove it in the future to a specific folder like you mentioned, but I would like to keep it organized and defined from the start. I would put it in the specific folder, but as you suggested, we can do that later on. If you would like to keep it like this, I am perfectly fine, and if you want to put it in a specific folder, I would also be perfectly fine (finer than in the first case 😄 )

Reply:
Moved class into services folder within core as I started working on #4951